tests/test_outputs.py

```diff
@@ -203,3 +206,18 @@ def test_results_str(test_output_results, test_str, request) -> None:
     test_str = request.getfixturevalue(test_str)
 
     assert test_output_results.__str__() == test_str
+
+
+@pytest.mark.parametrize("result_class", [RATapi.Results, RATapi.BayesResults])
```

Comment:
Maybe flip the order so we test that the correct result is returned irrespective of class i.e. `@pytest.mark.parametrize("result_class", [RATapi.BayesResults, RATapi.Results])`

Reply:
I'm not sure what you mean here? The method of parameterising runs four tests matching the load methods from both classes with both the bayes and non-bayes examples. What additional test would you like me to run?

Reply:
Sorry I missed that thought it was only running the two